freqtrade/plot/plotting.py

```diff
@@ -359,16 +360,23 @@ def load_and_plot_trades(config: Dict[str, Any]):
 
         store_plot_file(fig, filename=generate_plot_filename(pair, config['ticker_interval']),
                         directory=config['user_data_dir'] / "plot")
+        plot_data[generate_plot_filename(pair,
+                                         config['ticker_interval'])] = {"fig": fig, 
+                                                                        "data": dataframe, 
+                                                                        "trades": trades}
 
     logger.info('End of plotting process. %s plots generated', pair_counter)
+    logger.info(f'fig, data, trades are available with the keys {list(plot_data)}')
+    return plot_data
```

Comment:
Returning this here is pointless, if you use `generate_candlestick_graph()` directly in the notebook (as i said multiple times before) then this is not necessary.

You're reloading **everything** from file, if that's the intend, then you dont' need a notebook but can run the command as cli command as well.